blockchain/reactor_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	cmn "github.com/tendermint/tendermint/libs/common"
 	dbm "github.com/tendermint/tendermint/libs/db"
 	"github.com/tendermint/tendermint/libs/log"
+	memplmock "github.com/tendermint/tendermint/mempool/mock"
```

Comment:
I've seen this several times but never understood why we alias  mocks if there are no name clashes/conflicts. Couldn't this do without the alias. Then uses of this would read `mock.Mempool`.

Reply:
Do you think it would make sense to move all the mocks into one top-level package?

Reply:
Might be worth thinking about this. Not sure if that would be a good idea. But no, I only meant that we do not need aliases when there are no conflicts. No big thing tho.